src/client/js/scenes/MatchScene.js

```diff
@@ -55,42 +56,43 @@ export default class MatchScene extends FadeableScene {
     const websocketUrl = window.location.hostname === 'localhost'
       ? 'ws://localhost:2567'
       : 'wss://api' + window.location.hostname.substring(3) + ':443';
-    this.client = new Colyseus.Client(websocketUrl);
-    this.room = this.client.join('my_room');
-    this.room.onJoin.add(() => {
-      this.room.state.players.onAdd = (player, key) => {
-        let playerObj = new Player(this.add.sprite(player.pos_x, -1 * player.pos_y, 'cultist_blue'));
 
-        if (key === this.room.sessionId) {
-          this.player = playerObj;
-          this.cameras.main.startFollow(this.player.sprite);
-        }
+    var client = new Colyseus.Client(websocketUrl);
+    var room = client.join('my_room');
```

Comment:
please change room and client to object variables (this.room, this.client). It is used to make sure we disconnect from colyseus properly on the Leave Game button.